src/WebJobs.Extensions.DurableTask/Bindings/EntityTriggerAttributeBindingProvider.cs

```diff
@@ -53,6 +53,8 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 name = method.GetCustomAttribute<FunctionNameAttribute>()?.Name ?? method.Name;
             }
 
+            name = name.ToLower();
```

Comment:
Is this `ToLower()` necessary? I think function names are already stored in case-insensitive dictionaries.

Reply:
This was added after the initial changes broke some tests. We needed to allow for case insensitivity on the entity trigger as well. @ConnorMcMahon